src/metabase/driver/sql_jdbc/sync/describe_database.clj

```
@@ -96,13 +102,13 @@
                       (pr-str table-name)))
       true
       (catch Throwable e
-        (let [allow? (driver/query-canceled? driver e)]
-          (log/info (if allow?
-                      "%s: Assuming SELECT privileges: caught timeout exception"
-                      "%s: Assuming no SELECT privileges: caught exception")
-                    (str (when table-schema
-                           (str (pr-str table-schema) \.))
-                         (pr-str table-name)))
+        (let [allow? (driver/query-canceled? driver e)
+              table-str (str (when table-schema
+                               (str (pr-str table-schema) \.))
+                             (pr-str table-name))]
+          (if allow?
+            (log/infof "%s: Assuming SELECT privileges: caught timeout exception" table-str)
+            (log/infof e "%s: Assuming no SELECT privileges: caught exception" table-str))
```

Why do you think it is necessary to print exception on info level?

---

In the positive branch of the conditional, we know it is safe to ignore the exception without losing context because it’s a timeout. In the negative branch, we assume the exception is related to missing SELECT privileges, but we can’t be certain. The bug report that prompted this change showed that not logging the exception made it harder to debug; we had to rely on assumptions instead of concrete information.

As for why it's logged at the info level instead of debug or error, I followed the original behavior that was in place before [a refactor](https://github.com/metabase/metabase/commit/93bd0995b90696bf0400f9710749f1f65fb95e3b#diff-1e7eacafc3c5d4b4da2a64cd300aef24460b9ffc170c84a7841dc7b352312111L60-L104). That change, I believe unintentionally, caused the exception to be swallowed and the format string to go unused. This update simply restores the earlier, more transparent behavior.